README.md

```diff
@@ -0,0 +1,42 @@
+# Paste HTML to govspeak
+
+Converts HTML formatted rich content to [govspeak][] format (a markdown extension library for government editors) when pasted from clipboard into a form input or textarea.
+
+## Installation
+
+Add the compiled `/dist/paste-html-to-markdown.js` to your application assets.
+
+An npm installation will be available at a future point.
+
+## Usage
+
+```js
+element.addEventListener('paste', window.pasteHtmlToGovspeak)
+```
+
+## Browser support
+
+- Chrome
+- Firefox
+- Safari
+- Internet Explorer 11
+- Microsoft Edge
+
+[govspeak]: https://github.com/alphagov/govspeak
+
+## Development
+
+```
+npm install
+npm test
+```
+
+To continuously build files while developing run:
+
+```
+npm run watch
+```
+
+## License
+
+Distributed under the MIT license. See LICENSE for details.
```

Comment:
I think you'll need to drop a LICENSE file in too. Can just copy this and change the year: https://github.com/alphagov/content-publisher/blob/master/LICENCE

Reply:
Yep, added.